crates/uv/tests/build.rs

```diff
@@ -1147,6 +1205,135 @@ fn workspace() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn build_all_with_failure() -> Result<()> {
+    let context = TestContext::new("3.12");
+    let filters = context
+        .filters()
+        .into_iter()
+        .chain([
+            (r"exit code: 1", "exit status: 1"),
+            (r"bdist\.[^/\\\s]+-[^/\\\s]+", "bdist.linux-x86_64"),
+            (r"\\\.", ""),
+        ])
+        .collect::<Vec<_>>();
+
+    let project = context.temp_dir.child("project");
+
+    let pyproject_toml = project.child("pyproject.toml");
+    pyproject_toml.write_str(
+        r#"
+        [project]
+        name = "project"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = ["anyio==3.7.0"]
+
+        [tool.uv.workspace]
+        members = ["packages/*"]
+
+        [build-system]
+        requires = ["setuptools>=42"]
+        build-backend = "setuptools.build_meta"
+        "#,
+    )?;
+
+    project.child("src").child("__init__.py").touch()?;
+    project.child("README").touch()?;
+
+    let member_a = project.child("packages").child("member_a");
+    fs_err::create_dir_all(member_a.path())?;
+
+    let member_b = project.child("packages").child("member_b");
+    fs_err::create_dir_all(member_b.path())?;
+
+    member_a.child("pyproject.toml").write_str(
+        r#"
+        [project]
+        name = "member_a"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = ["iniconfig"]
+
+        [build-system]
+        requires = ["setuptools>=42"]
+        build-backend = "setuptools.build_meta"
+        "#,
+    )?;
+
+    member_a.child("src").child("__init__.py").touch()?;
+    member_a.child("README").touch()?;
+
+    member_b.child("pyproject.toml").write_str(
+        r#"
+        [project]
+        name = "member_b"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = ["iniconfig"]
+
+        [build-system]
+        requires = ["setuptools>=42"]
+        build-backend = "setuptools.build_meta"
+        "#,
+    )?;
+
+    member_b.child("src").child("__init__.py").touch()?;
+    member_b.child("README").touch()?;
+
+    // member_b build should fail
+    member_b.child("setup.py").write_str(
+        r#"
+        from setuptools import setup
+
+        setup(
+            name="project",
+            version="0.1.0",
+            packages=["project"],
+            install_requires=["foo==3.7.0"],
+        )
+        "#,
+    )?;
+
+    // Build all the packages
+    uv_snapshot!(&filters, context.build().arg("--all").arg("--no-build-logs").current_dir(&project), @r###"
+    success: false
+    exit_code: 2
+    ----- stdout -----
+
+    ----- stderr -----
+    Building source distribution...
+    Building source distribution...
+    Building source distribution...
```

Comment:
I think we should prefix these with the member name (like `[member_a] Building source distribution...`).